test/Confluent.Kafka.UnitTests/Consumer.cs

```diff
@@ -40,6 +40,18 @@ public void Constuctor()
             e = Assert.Throws<ArgumentException>(() => { var c = new Consumer<Null, string>(config, null, new StringDeserializer(Encoding.UTF8)); });
             Assert.True(e.Message.Contains("group.id"));
 
+            e = Assert.Throws<ArgumentException>(() => 
+            {
+                var validConfig = new Dictionary<string, object>
+                {
+                    { "bootstrap.servers", "localhost:9092" },
+                    { "group.id", "my-group" }
+                };
+                var deserializer = new StringDeserializer(Encoding.UTF8);
+                var c = new Consumer<string, string>(validConfig, deserializer, deserializer); 
```

Comment:
I put this check in because I saw you doing this in the avro PR... It's an error because the Consumer calls Configure on both serializers, and furthermore this call takes a bool parameter indicating whether it's a value of key serializer. Most serializers do nothing in the Configure method, so it will work fine, but in the avro case it won't.